src/NodeServices/Interfaces.cs

```diff
@@ -150,7 +166,7 @@ internal interface IExtensionApplication
         /// <summary>
         /// Called when first time this application is loaded.
         /// </summary>
-        void StartUp();
+        void StartUp(ExtensionStartupParams startParams);
```

Comment:
breaking change ....but should be ok since the interface is internal.
Only LibG implements this interface so we can make the necessary changes to that specific implementation class to keep API compatibility